packages/vuetify/src/components/VSheet/VSheet.ts

```diff
@@ -1,60 +1,86 @@
 // Styles
 import './VSheet.sass'
 
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeBorderRadiusProps, useBorderRadius } from '@/composables/border-radius'
+import { makeDimensionProps, useDimension } from '@/composables/dimensions'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeTagProps } from '@/composables/tag'
+import { useTheme } from '@/composables/theme'
+
 // Utilities
-import { defineComponent, h } from 'vue'
+import { computed, defineComponent, h } from 'vue'
 import makeProps from '@/util/makeProps'
+import propsFactory from '@/util/propsFactory'
 
-// Composables
-import {
-  makeBorderProps,
-  makeBorderRadiusProps,
-  makeDimensionProps,
-  makeElevationProps,
-  makePositionProps,
-  makeTagProps,
-  useBorder,
-  useBorderRadius,
-  useDimension,
-  useElevation,
-  usePosition,
-  useTheme,
-} from '@/composables'
+// Types
+import type { BorderProps } from '@/composables/border'
+import type { BorderRadiusProps } from '@/composables/border-radius'
+import type { DimensionProps } from '@/composables/dimensions'
+import type { ElevationProps } from '@/composables/elevation'
+import type { PositionProps } from '@/composables/position'
+import type { TagProps } from '@/composables/tag'
+
+export interface SheetProps extends
+  BorderProps,
+  BorderRadiusProps,
+  DimensionProps,
+  ElevationProps,
+  PositionProps,
+  TagProps {}
+
+export const makeSheetProps = propsFactory({
+  ...makeBorderProps(),
+  ...makeBorderRadiusProps(),
+  ...makeDimensionProps(),
+  ...makeElevationProps(),
+  ...makePositionProps(),
+  ...makeTagProps(),
+})
+
+export function useSheet (props: SheetProps, name: string) {
+  const { themeClasses } = useTheme()
+  const { borderClasses } = useBorder(props)
+  const { borderRadiusClasses } = useBorderRadius(props)
+  const { dimensionStyles } = useDimension(props)
+  const { elevationClasses } = useElevation(props)
+  const { positionClasses, positionStyles } = usePosition(props, name)
+
+  const sheetClasses = computed(() => {
+    return [
+      name,
```

Comment:
`name` seems a bit vague, could it be `componentName` maybe or `classPrefix` or something more descriptive?

Reply:
After the meeting today we decided to remove this and refactor later if it makes sense.